rpc/client/client_test.go

```diff
@@ -214,6 +215,77 @@ func TestGetBlock(t *testing.T) {
 	require.NoError(err)
 }
 
+func TestGetCommit(t *testing.T) {
+	require := require.New(t)
+	assert := assert.New(t)
+	mockApp, rpc := getRPC(t)
+	mockApp.On("BeginBlock", mock.Anything).Return(abci.ResponseBeginBlock{})
+	mockApp.On("Commit", mock.Anything).Return(abci.ResponseCommit{})
+
+	block := getRandomBlock(1, 5)
```

Comment:
This test should be improved. You should store more than one block and try to retrieve commits for all of the blocks, and for `nil` height.